poetry.lock

```diff
@@ -0,0 +1,144 @@
+[[package]]
```

Comment:
As we discussed, no issues with you using whatever build process you like as long as the standard documented version works ok